js/okx.js

```diff
@@ -1605,13 +1613,18 @@ module.exports = class okx extends Exchange {
         await this.loadMarkets ();
         const market = this.market (symbol);
         const price = this.safeString (params, 'price');
+        const isHKTime = this.safeString (params, 'isHkTime');
```

Comment:
I would move that to `options` instead of using `params` here.
Something like `exchange.options['fetchOHLCV']['timezone'] = 'UTC' // or HK`
I will rewrite this before merging.

Reply:
Moving this to `options` seems better. I rewrote it so you can adjust it if needed